src/clj/rems/db/attachments.clj

```diff
@@ -1,9 +1,13 @@
 (ns rems.db.attachments
```

Comment:
`rems.db.attachments` is probably not the right choice of namespace. Malware scanner should work similarly to `rems.multipart/size-limiting-temp-file-store`, which checks uploaded file (as it is being uploaded) and returns contextually appropriate error code if the file size exceeds configured limit. Neither database or service layer should be tied to implementation details. Perhaps multipart function could be extended to handle malware scanner as well.

Reply:
fixed in [60e8d42](https://github.com/CSCfi/rems/pull/3202/commits/60e8d42d2de6226b822912278060434bbc063343)